package.json

```diff
@@ -33,27 +33,28 @@
     "slick-carousel": "1.6.0",
     "snyk": "1.19.1",
     "webpack": "1.13.2",
-    "webpack-stream": "3.2.0"
-  },
-  "devDependencies": {
+    "webpack-stream": "3.2.0",
+
     "browser-sync": "2.17.5",
-    "chai": "3.5.0",
     "glob-all": "3.0.1",
+    "gulp-imagemin": "3.1.1",
+    "gulp-eslint": "3.0.1",
+    "gulp-concat": "2.6.0",
+    "gulp-rename": "1.2.2",
     "gulp-autoprefixer": "3.1.1",
     "gulp-clean-css": "2.3.2",
-    "gulp-concat": "2.6.0",
-    "gulp-coveralls": "0.1.4",
-    "gulp-eslint": "3.0.1",
     "gulp-header": "1.8.8",
-    "gulp-imagemin": "3.1.1",
-    "gulp-istanbul": "1.1.1",
     "gulp-less": "3.3.0",
-    "gulp-mocha": "3.0.1",
-    "gulp-plumber": "1.1.0",
-    "gulp-rename": "1.2.2",
     "gulp-sourcemaps": "2.1.1",
-    "gulp-util": "3.0.7",
+    "gulp-coveralls": "0.1.4",
+    "gulp-istanbul": "1.1.1",
+    "gulp-mocha": "3.0.1",
     "is-reachable": "1.3.0",
+    "gulp-util": "3.0.8"
+  },
+  "devDependencies": {
+    "chai": "3.5.0",
+    "gulp-plumber": "1.1.0",
     "jasmine-reporters": "2.1.1",
     "jasmine-spec-reporter": "2.4.0",
     "jquery": "1.11.3",
```

Comment:
`jquery` should get moved to dependencies. We still require it for a CDN fallback in `base.html`.